modules/workspaces/variables.tf

```diff
@@ -71,6 +71,12 @@ variable "vcs_repo" {
   type = map(string)
 }
 
+variable "vcs_branch" {
```

Comment:
This seems to be a duplicate variable that would already exist in `vcs_repo`. Reasoning?

Reply:
`vcs_repo.branch` provides the default (`master` typically). `vcs_branch` allows the user to override the configuration for the workspace to a different branch (e.g. `develop`). This is part of a multi-branch terraform repo approach that I was convinced to try out for a client. It's still up in the air if this will be the long term approach, but regardless if someone wants to do a multi-branch workflow via this module then this would be needed.

Reply:
So which one wins? If we specify a branch from `vcs_repo.branch` and `vcs_branch`, which one is ultimately the choice?

Reply:
@johncblandii good question -- See https://github.com/cloudposse/terraform-tfe-cloud-infrastructure-automation/pull/6/files#diff-3f3dffc99a63b175df766e974417a446e0a644b28269cfa6b08f2db755905539R18. The vcs_branch provided via the stack is the override. I figured that was clear, but if you're asking the question... maybe we should rename to `vcs_branch_override` instead? What do you think?

Reply:
That's where I was going with that thought. The variable should be clear upfront.

Reply:
Gotcha -- I'll rename 👍

Reply:
Updated!